components/collections/codepointtrie_builder/src/lib.rs

```diff
@@ -99,12 +103,73 @@ where
     /// Build the [`CodePointTrie`].
```

Comment:
It's not used when calling `build()` though.

Reply:
I think a better model could be to make the executable an Option field, make `build()` use it if available, and otherwise fall back to wasm. Basically take the stuff that @Manishearth wrote in datagen and move it into icu_codepointtrie_builder directly.

Reply:
I'm down to do that. What's the behavior when Wasm isn't available? Should I create an error type for this crate so that e.g. datagen can display a "please build with wasm" message?